stream/src/main/scala/org/apache/pekko/stream/ActorMaterializer.scala

```diff
@@ -223,7 +223,7 @@ private[pekko] object ActorMaterializerHelper {
 abstract class ActorMaterializer extends Materializer with MaterializerLoggingProvider {
 
   @deprecated(
-    "Use attributes to access settings from stages, see https://doc.akka.io/docs/akka/2.6/project/migration-guide-2.5.x-2.6.x.html",
+    "Use attributes to access settings from stages, see https://pekko.apache.org/docs/pekko/2.6/project/migration-guide-2.5.x-2.6.x.html",
```

Comment:
we have removed the akka migration guides - so best to revert this change so it links to the akka docs

Reply:
all other changes like this should be reverted too